FRINXio/frinx-frontend
# Fix executed wf bug

| Q                        | A 
| ------------------------ | ---
| Fixed Issue?             | `FD-483` 
| Tests Added + Pass?      | Yes
| Any Dependency Changes?  |

https://frinxhelpdesk.atlassian.net/browse/FD-483?atlOrigin=eyJpIjoiNzQ0ZDYzYmQ5MGU5NDUxZGFkMTRkNjkwYTRmMjliNDMiLCJwIjoiaiJ9

In executed workflows > Edit & run tab the inputs werent displaying values of inputs, neither they could be changed.

HOW TO TEST:

Select any executed workflow and head to edit & run tab, try to change input values and hit Execute button. Values should be displayed and saved.
